client/src/components/MapContainer.js

```diff
@@ -66,13 +64,11 @@ class MapContainer extends Component {
           containerElement={<div style={{ height: '400px' }} />}
           mapElement={<div style={{ height: '100%' }} />}
           markers={this.state.markers}
-
-
           onMarkerClick={this.handleMarkerClick}
           onInfoClose={this.handleInfoClose}
           defaultCenter={this.props.defaultCenter}
           defaultZoom={this.props.defaultZoom}
-
+          onPopClick={this.props.handlePopupClick}
```

Comment:
Perhaps this should be called `onPopupClick` to match the handler function name?

Reply:
Done.